components/Trailer.vue

```diff
@@ -1,22 +1,288 @@
+        .to(
+          '.play-circle-01',
+          {
+            duration: 0.7,
+            opacity: 0.1,
+            rotation: '+=360',
+            strokeDasharray: '456 456',
+            ease: 'power1.inOut',
+          },
+          0
+        )
+        .to(
+          '.play-circle-02',
+          {
+            duration: 0.7,
+            opacity: 0.1,
+            rotation: '-=360',
+            strokeDasharray: '411 411',
+            ease: 'power1.inOut',
+          },
+          0
+        )
+    },
+    configOpen() {
+      this.openTL = this.$gsap
+        .timeline({ paused: true })
+        .to(
+          '.play-backdrop',
+          {
+            duration: 0.8,
+            opacity: 0,
+            visibility: 'visible',
+            ease: 'power2.inOut',
+          },
+          0
+        )
+        .to(
+          '#wrapper',
+          {
+            duration: 1.2,
+            backgroundColor: '#fff',
+            height: '500px',
+            ease: 'power2.inOut',
+          },
+          0
+        )
+        .to(
+          '.play-close',
+          {
+            duration: 0.8,
+            opacity: 1,
+            ease: 'power2.inOut',
+          },
+          0
+        )
+        .to(
+          '.play-perspective',
+          {
+            duration: 0.8,
+            xPercent: 0,
+            scale: 1,
+            ease: 'power2.inOut',
+          },
+          0
+        )
+        .to(
+          '.play-triangle',
+          {
+            duration: 0.8,
+            scaleX: 1,
+            ease: 'expoScale(10,0.01, power2.inOut)',
+          },
+          0
+        )
+        .to(
+          '.play-triangle',
+          {
+            duration: 1,
+            rotationY: 0,
+            ease: 'expoScale(10,0.01, power2.inOut)',
+          },
+          0
+        )
+        .to(
+          '.play-video',
+          {
+            duration: 1,
+            visibility: 'visible',
+            opacity: 1,
+          },
+          0.5
+        )
+    },
+    playVideo() {
+      this.openTL.play()
+    },
+    closeVideo() {
+      this.openTL.reverse()
+    },
+    rotateCircles() {
+      this.rotateTL.play()
+    },
+    rotateCirclesRev() {
+      this.rotateTL.reverse()
+    },
+  },
+  mounted() {
+    this.positionCircles()
+    this.configOpen()
+    this.configRotate()
+  },
 }
 </script>
+<style scoped>
+#wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  /* height: 100vh; */
+  height: 400px;
+  margin: 0;
+  background-color: #333;
+  overflow: hidden;
+}
+
+.play-button {
+  width: 152px;
+  height: 152px;
+  position: relative;
+  cursor: pointer;
+}
+
+.play-backdrop {
+  width: 600px;
+  height: 500px;
+  position: fixed;
+  /* left: 0;
+  top: 0; */
+  background-color: #000;
+  opacity: 0;
+  visibility: hidden;
+}
+
+.play-close {
+  width: 30px;
+  height: 30px;
+  position: absolute;
+  right: 15px;
+  bottom: calc(100% + 15px);
```

Comment:
this won't be possible with tailwind but you can use inline css instead that will be less bloated